packages/react/src/primitives/Rating/__test__/Rating.test.tsx

```diff
@@ -55,8 +55,8 @@ describe('Rating', () => {
 
     const emptyIcon = container.getElementsByClassName(
       'amplify-rating-icon-empty'
-    )[0];
-    expect((emptyIcon['style'] as React.CSSProperties).color).toBe('red');
+    )[0] as HTMLElement;
+    expect((emptyIcon.style as React.CSSProperties).color).toBe('red');
```

Comment:
Do we still need to cast here now that `emptyIcon` is casted?

Reply:
Good catch, I'll remove it

Reply:
updated